.github/workflows/c-cpp.yml

```diff
@@ -2,9 +2,9 @@ name: C/C++ CI
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "support_openssl_3.1*" ]
```

Comment:
Why change the branch from main to support_openssl_3.1*?

Reply:
Yes, that makes sense.
I'll update the PR.